javatools_backend/.dir-locals.el

```diff
@@ -0,0 +1,6 @@
+;; List of lists
+(
```

Comment:
While now added to .gitignore, this will still cause the elisp file to be merged to master. Please delete it with git rm, commit, and put the original file back in the same location, and do a "git status" to ensure that this doesn't show up as under source control but is still in the file system.

Please do this for the other elisp files too.